fhir-audit/src/main/java/com/ibm/fhir/audit/configuration/handlers/ComponentIpHandler.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.audit.configuration.handlers;
+
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.StringJoiner;
+import java.util.logging.Logger;
+
+import com.ibm.fhir.config.FHIRConfigHelper;
+import com.ibm.fhir.config.FHIRConfiguration;
+
+/**
+ * Calculates the Component Ip and subsequent retrieval of the IPs
+ */
+public class ComponentIpHandler {
```

Comment:
Please see FHIRException.getUniqueId() and determine if there is common functionality that we can use.  At least we should be using the same approach in the two places I think.